synapse/storage/data_stores/main/room.py

```diff
@@ -58,6 +58,19 @@ class RoomSortOrder(Enum):
 
     ALPHABETICAL = "alphabetical"
```

Comment:
Please add a comment above `ALPHABETICAL` saying that alphabetical and size are deprecated for name and joined_members. Thanks!